tests/frontends/test_pytorch_ops.py

```diff
@@ -130,12 +147,19 @@ def mksens(x):
     res = gpipeline.run(input=fn, argspec=[*argspec, sens_type])
 
     myia_grads = res['output'](*args, sens)
-    assert eqtest(pytorch_grads, myia_grads, rtol=1e-05, atol=1e-06)
+    assert eqtest(pytorch_grads, myia_grads, rtol=grad_rtol, atol=grad_atol)
 
 
 @myia_function_test(marks=[pytest.mark.grad], id='grad')
-def fwd_and_bwd(self, fn, args, broad_specs=None, pipeline=standard_pipeline):
-    _fwd_and_bwd(fn, args, broad_specs, pipeline)
+def _fwd_and_bwd(self, fn, args, broad_specs=None, pipeline=standard_pipeline,
+                 backend=False, atol=1e-8, rtol=1e-5,
+                 grad_atol=1e-6, grad_rtol=1e-5):
+    __fwd_and_bwd(fn, args, broad_specs, pipeline, backend, atol=atol,
+                  rtol=rtol, grad_atol=grad_atol, grad_rtol=grad_rtol)
```

Comment:
Inline `__fwd_and_bwd` in here, since it's the only use.

Reply:
done